contracts/TreasureMarketplace.sol

```diff
@@ -20,6 +21,9 @@ import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
 contract TreasureMarketplace is OwnableUpgradeable, PausableUpgradeable, ReentrancyGuardUpgradeable {
     using SafeERC20Upgradeable for IERC20Upgradeable;
 
+    bytes4 private constant INTERFACE_ID_ERC721 = 0x80ac58cd;
```

Comment:
Can move these to below next to other constants.

Reply:
Makes sense